devices/stm32/stm32f1-03-c_d_e.xml

```diff
@@ -141,6 +137,96 @@
     </driver>
     <driver name="usb" type="stm32-v1.1"/>
     <driver name="wwdg" type="stm32-v1.0"/>
+    <driver name="dma" type="stm32-channel">
+      <channels>
```

Comment:
This `<channels>` does not know to which instance it belongs.

https://github.com/modm-io/modm-devices/blob/d02739c9585e03f84b4693446b0654202dd53623/devices/stm32/stm32f1-03-c_d_e.xml#L227-L228

@salkinium

Reply:
Thanks, fixed!